book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -8,24 +8,324 @@ This can involve hosting different events that are attended by people with lived
+By including contact details and explaining the roles of organizers, it can help attendees feel understood and welcomed instead of being redirected to multiple people who are unable to help them. 
+
+If possible, it can also be helpful to have a dedicated person who can be easily located and contacted during the event that is able to take action if there is a problem. 
+For example, if an elevator is broken, the restroom blocked, accessible doors not opening, or the audio induction loop not working. 
+This person could be available during event registration or at the start of the event to introduce themselves and provide assistance to those who need it. 
+This may not be feasible for all events and may be more important for in-person events, but it is important to consider how to make the event accessible and inclusive throughout. 
+Inclusion can be planned for, but it also needs implementation and support to make it happen.
+
+
+(cl-inclusive-events-planning-post-event)=
+## After the Event
+Consider if you are able to make content from the event accessible to attendees, or open to a wider audience, after the event. 
+
+For events that have recorded sessions, this may be uploading the content to YouTube and ensuring that the captions have been reviewed and added to any video content. 
+Having a human reviewer of captions can be helpful in catching any errors made by automatic captioning software and ensuring that content is of a good quality (such as editing out any audio glitches and flagging triggers for photosensitive epilepsy). 
+
+If your event included content such as slides or documented notes, consider where and how you will archive them so that they can be accessed and referenced by attendees. 
+Online repositories such as Zenodo are commonly used by research communities but are not universally design-compliant and require both technical expertise and knowledge of how to use and access it as well as requiring an internet connection to access content. 
+Consider using Zenodo in addition to an accessible format. 
+This could be a website commonly visited by your attendees, sending post-event summary information to attendees, or providing recordings of sessions that display session content.
+
+Finally, for in-person events and related to safeguarding, consider how you will ensure that everyone is able to access return transport safely. 
+This is particularly important to consider for attendees with lived experience and any disabilities that may make public transport more difficult to access. 
+Although your event may have formally finished, are there support pathways you can put in place such as arranging a coach or taxis to a train station, or by ending an event within daylight hours to ensure that attendees are not travelling in the dark? 
```

Comment:
```suggestion
Even after the event has formally concluded, make sure you are familiar with support options such as arranging coaches or taxis to a train station. Additionally, consider scheduling the event to end within daylight hours to enhance attendee safety and avoid travel in the dark.
```

Reply:
Hi @JFormoso!

Thank you for the suggestions, they're all really helpful. 

When you're making inline edits, would you be able to put each sentence on a new line?

Even while drafting, it's really helpful to align work to The Turing Way's [Style Guide](https://the-turing-way.netlify.app/community-handbook/style.html) 😄